frontend/src/metabase-types/api/task.ts

```diff
@@ -9,6 +9,7 @@ export interface Task {
   ended_at: string;
   task: string;
   task_details: Record<string, unknown> | null;
+  status: "success" | "started" | "failed" | "unknown";
```

Comment:
Am I right that "unknown" is only expected for historical records before this field was added? Perhaps worth noting that here.

Reply:
added one in 891e6cd6a7